src/menu/components/boxart.c

```diff
@@ -65,7 +65,7 @@ void component_boxart_free (component_boxart_t *b) {
 }
 
 void component_boxart_draw (component_boxart_t *b) {
-    if (b && b->image && b->image->width == BOXART_WIDTH && b->image->height == BOXART_HEIGHT) {
+    if (b && b->image) {
```

Comment:
Given that the check for the exact size is now ignored, please can you check what effect this will cause, and if necessary add a Max size for width and height.